app/controllers/foreman_bootdisk/api/v2/disks_controller.rb

```diff
@@ -48,6 +51,12 @@ def host
 
         private
 
+        def action_is_allowed(action = params[:action])
+          return true if Setting.find_by_name('bootdisk_allowed_types').value.include? action
```

Comment:
There is the exactly same code in the `subnet_disks_controller.rb` so it leads to extract this method to a concern probably. However, this only one method so I don't know if it's worth it.

Reply:
Isn't writing a helper method in settings/bootdisk.rb more simple?

Reply:
yeah, that would probably also work and logically belongs there, :+1: